docs/migration/README.rst

```diff
@@ -18,6 +18,66 @@ This migration combines two migrations into one:
 Performing both migrations together reduces overall work, though you need to
 perform more steps before seeing results.
 
+Ensure code repo is visible
+---------------------------
+
+App SRE only supports fetching code without authentication.  This means that
+code must either be a public repo on Github or on gitlab.cee.redhat.com.  If
+your code repo is currently private on Github, it needs to either be made public
+or moved to Gitlab.  If your repo is already public on Github or Gitlab, then
+you're good to go.
+
+Open Sourcing a Project
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you choose to keep your project on Github, it must be open sourced.  This
+usually entails several things:
+
+#. Adding a license file to the root of your repository
+#. Adding a reference to the license at the top of each source file
+#. Reviewing the repo's commit history for sensitive information
+
+Moving a Project from Github to Gitlab
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Moving a project from Github to Gitlab should be relatively straightforward, but
+will require some workflow changes for devs.
+
+#. Create a new repo on gitlab.cee.redhat.com.  There is already an
+   `insights-platform`_ group that can be used to house repos, but other groups
+   can be used if desired.
+#. Change your local upstream to gitlab.  Example::
+
+      git remote set-url upstream git@gitlab.cee.redhat.com:insights-platform/new-repo.git
+
+You shouldn't need to worry about existing CI integrations since the goal of
+this migration is to have all CI integrated into app-interface.
+
+.. _insights-platform: https://gitlab.cee.redhat.com/insights-platform/
+
+Why?
+^^^^
+
+The technical reason private Github repositories are not supported is because
+Jenkins jobs managed by App SRE do not support making authenticated pulls to git
+repos.  While support for this can of course be added, no team has been able to
+justify the work to add this support.
+
+If a dev team is not comfortable making their code public to the world, then
+their interests would be better served by moving to Gitlab since it is of course
+hosted within the Red Hat internal network, thus reducing the risk of the code
+leaking publicly.
+
+If a dev team *is* comfortable making their code public, then they should go
+ahead and schedule the work to open source their app.  If this work is too
+costly, then their app should be hosted on Gitlab until the open sourcing work
+can be completed.
+
+Lastly, the Clouddot org has consistently struggled to reduce the head count
```

Comment:
I'd suggest making this paragraph a stronger stance.  Something like:
`Lastly, the Clouddot org's expectation is that private repositories live in Gitlab.  If a project is open source, it should live in Github public repositories.  By reserving Github repositories for open source projects we can better manage the number of Github licenses required.  `

While it hasn't been strongly enforced, the organizations position is that private repositories live in Gitlab and public ones live in Github.